src/modal/test/modal.spec.js

```diff
@@ -1064,12 +1064,12 @@ describe('$uibModal', function() {
 
     describe('controller', function() {
       it('should accept controllers and inject modal instances', function() {
-        var TestCtrl = function($scope, $uibModalInstance) {
```

Comment:
This test should not be repurposed - two separate tests should be added, one to test the default value and one to test that it can be overridden.